docs/docs/dev/policies/mesh.md

```diff
@@ -108,6 +113,8 @@ metadata:
 spec:
   ...
 ```
+
+Kuma consumes all [Policies](/policies) on all namespaces and joins each to an individual `Mesh`, identified by this property.
```

Comment:
Interesting, the original issue https://github.com/kumahq/kuma-website/issues/178 says it fetches policies from all namespaces, but they really don't exist in a namespace. The idea is that they are global. That makes sense because if they were in a particular namespace you could have duplicates when you pulled them all in to kuma-cp.